themes/gatsby-theme-catalyst-bery/src/components/post-footer.js

```diff
@@ -8,7 +8,8 @@ const PostFooter = ({ previous, next }) => {
   const rootPath = sanityPostPath.replace(/\/*$/, `/`) //Ensure trailing slash
   return (
     <div sx={{ mb: 3 }}>
-      <Styled.hr sx={{ bg: "textGray" }} />
+      <Styled.hr sx={{ bg: "textGrayLight" }} />
+      <Styled.h3>More Recent Posts</Styled.h3>
```

Comment:
Maybe bump the font size down a little bit? What do you think? Feels a bit big to me?